packages/templates/src/raw/travel.json

```diff
@@ -1,4 +1,12 @@
 {
+  "current": "22081c00-a02f-4085-84de-0f4a31253222",
+  "selection": ["7dae1bd9-9f88-4ae8-93eb-f3d908cace09"],
```

Comment:
🤔 

Wouldn't that cause unexpected selection when a user tries to use a template?

Reply:
my bad, grabbed the updated template while selection was present. fixed.